typings/pluginapi.d.ts

```diff
@@ -156,6 +243,11 @@ export interface PluginConfig {
    * All plugin logs should be logged via this logger.
    */
   readonly logger: Logger
+
+  /**
+   * Plugins should default to this directory when applicable.
```

Comment:
I wasn't sure how to word this better, definitely open to suggestions. 

For example, for the Jupyter plugin this means launching Jupyter in this directory so when it opens that's the directory that shows up.

For other plugins it might mean something different though.

The difference from `process.cwd` is that it can be set by the user (`code-server working/directory`).

Reply:
Mind adding this comment ^ to your code comment? I think doing that will provide enough explanation. We can iterate on it later as needed.